.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Suggest an idea for Trino Python client
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+### Describe the feature
+
+<!-- A clear and concise description of what you want to happen.
+What kind of use case will this feature be useful for?
```

Comment:
I'm afraid when asked for a use case most people would just repeat the solution instead of defining the problem. We should ask for it explicitly.
```suggestion
<!-- A clear and concise description of what you want to happen
and what problem it would solve.
```